src/rust/cryptography-cffi/build.rs

```diff
@@ -25,6 +25,9 @@ fn main() {
     let out_dir = env::var("OUT_DIR").unwrap();
     // FIXME: maybe pyo3-build-config should provide a way to do this?
     let python = env::var("PYO3_PYTHON").unwrap_or_else(|_| "python3".to_string());
+    if let Ok(dir) = env::var("OPENSSL_DIR") {
```

Comment:
This should probably just be in rust-openssl right? Is there a reason it shouldn't?

Reply:
Nope! Up there would make the most sense, especially since it finds `OPENSSL_DIR` if the env isn't set anyway.